Bools  and Ghost-Gobble Arcade Game Typos and Revisions (#2498)
exercism/python: exercises/concept/ghost-gobble-arcade-game/.docs/instructions.md

# Instructions

In this exercise, you need to implement some rules from [Pac-Man][Pac-Man], the classic 1980s-era arcade-game.

You have four rules to implement, all related to the game states.

> > _Don't worry about how the arguments are derived, just focus on combining the arguments to return the intended result._

Reviewer: ```suggestion
> _Do not worry about how the arguments are derived, just focus on combining the arguments to return the intended result._
```

## 1. Define if Pac-Man eats a ghost

> Define the `eat_ghost()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man is able to eat the ghost.

Reviewer: ```suggestion
Define the `eat_ghost()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man is able to eat the ghost.
```

 The function should return `True` only if Pac-Man has a power pellet active and is touching a ghost.

```python
>>> eat_ghost(False, True)
...
False
```

## 2. Define if Pac-Man scores

> Define the `score()` function that takes two parameters (_if Pac-Man is touching a power pellet_ and _if Pac-Man is touching a dot_) and returns a boolean value if Pac-Man scored.

Reviewer: ```suggestion
Define the `score()` function that takes two parameters (_if Pac-Man is touching a power pellet_ and _if Pac-Man is touching a dot_) and returns a Boolean value if Pac-Man scored.
```

 The function should return `True` if Pac-Man is touching a power pellet or a dot.

```python
>>> score(True, True)
...
True
```

## 3. Define if Pac-Man loses

> Define the `lose()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man loses.

Reviewer: ```suggestion
Define the `lose()` function that takes two parameters (_if Pac-Man has a power pellet active_ and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man loses.
```

 The function should return `True` if Pac-Man is touching a ghost and does not have a power pellet active.

```python
>>> lose(False, True)
...
True
```

## 4. Define if Pac-Man wins

> Define the `win()` function that takes three parameters (_if Pac-Man has eaten all of the dots_, _if Pac-Man has a power pellet active_, and _if Pac-Man is touching a ghost_) and returns a boolean value if Pac-Man wins.

Reviewer: ```suggestion
Define the `win()` function that takes three parameters (_if Pac-Man has eaten all of the dots_, _if Pac-Man has a power pellet active_, and _if Pac-Man is touching a ghost_) and returns a Boolean value if Pac-Man wins.
```
